new/PRS13947Eleskndros.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13947Eleskndros" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ǝleskǝndros</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="VP"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="VP" when="2022-11-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
+                  <persName xml:lang="gez" >እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
+                  <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
+                  <persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>
+                    <faith type="Traditional">pagan</faith>
+                    <occupation type="ruler">Ruler during the Roman Empire</occupation>
+                    <birth/>
+                    <death/>
+                    <floruit>0303-0305</floruit>
```

Comment:
the right way to encode dates should be 
`<floruit notBefore="0303" notAfter="0305">Beginning of the 4th century; period of the 'Big Persecutions' against the Christians of the Roman Empire.</floruit>
`
are the dates correct?

Reply:
yes, this is the period when the big persecutions from Dioceltian occured. If this the question, yes, it is correct.